test-fixtures/updateBlob/02/expectedResult.json

```diff
@@ -0,0 +1,3 @@
+{
+  "Error": true
```

Comment:
Variable naming conventions should be uniform: camelCase these variables (there are also other within the test set)?

Reply:
`{"error": true}` is in the file for prevent file loading errors and if test does not work expected result does not match the output

Reply:
fixed camelCase